template/.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
+echo "type-checking..."
+./node_modules/.bin/tsc --noEmit
```

Comment:
> no new line at the end of line.

this warning appears in most of the files.

Reply:
How do you get this error? Could you write steps to reproduce?

Reply:
Github show this warning at the end of file. It may not be something important, just want to state if it is.